site/docs/v1/tech/apis/_import-users-request-body.adoc

```diff
@@ -103,7 +103,11 @@ The User's mobile phone number. This is useful is you will be sending push notif
 [field]#users``[x]``.password# [type]#[String]# [required]#Required#::
 The User's password. If [field]#encryptionScheme# is provided this value is the existing password hash, if [field]#encryptionScheme# is not provided this value is assumed to be plain text and it will be hashed during import.
 +
-[field]#password# should contain only the password hash as a base64 encoded string. Depending on how your password database stores hashes, additional processing may be required. Please see the <<Password Hashes>> section for more information about importing particular password hashes.
+[field]#password# should contain only the password hash.
```

Comment:
I think this is just a description of the field inside of the `users``[x]``.password` field def right? If so - it is a little confusing seeing it like to have it start with `[field]` since it is not a field. Maybe update it start with a word instead, such as:

> The value for [field]#password# should only contain the password hash. Unless....